pkgs/top-level/all-packages.nix

```diff
@@ -33779,6 +33779,11 @@ with pkgs;
   minetestclient = minetestclient_5;
   minetestserver = minetestserver_5;
 
+  minetestPackages = dontRecurseIntoAttrs (import ../games/minetest/packages {
+    callPackage = newScope minetestPackages;
+  });
+  minetestWithPackages = callPackage ../games/minetest/minetest-with-packages.nix { };
+
```

Comment:
What is the convention for *withPackages packages attribute path? Should it stay at top level? inside `minetestPackages`? inside `minetest`?